### ISSUE TITLE
Injury from hyperextension or extreme abduction

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9910]
**Preferred term label:**
Injury from hyperextension or extreme abduction

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Thumb ulnar collateral ligament injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
we would regard injuries as diseases and would use Mondo terms -- there is a new term tracker  [here](https://github.com/monarch-initiative/mondo)

**Comment by MickeySegal** [comment]
We clarified the finding / disease distinction and renamed our finding "Hyperextension or extreme abduction at a joint by history", which should get an HPO code.
